api/client/src/main/java/org/projectnessie/client/auth/oauth2/PasswordFlow.java

```diff
@@ -22,9 +22,9 @@
  * href="https://datatracker.ietf.org/doc/html/rfc6749#section-4.3">Resource Owner Password
  * Credentials Grant</a> flow.
  */
-class ResourceOwnerPasswordFlow extends AbstractFlow {
+class PasswordFlow extends AbstractFlow {
 
-  ResourceOwnerPasswordFlow(OAuth2ClientConfig config) {
+  PasswordFlow(OAuth2ClientConfig config) {
```

Comment:
Renamed, merely for consistency with other existing classes belonging to the `password` grant type.